iRob_Interface:
    ros__parameters:
        serial_port:            "/dev/iRob"
        robot_frame_id:         "base_link"
        mouse_odom_frame_id:    "odom"
        mag_sensor_topic:       "imu/mag"
        imu_sensor_topic:       "imu/data_raw"
        mouse_x_constant:       0.03
        mouse_y_constant:       0.03
        gyro_constant:          0.001221731
        mag_constant:           0.08075
        accel_constant:         0.0196133
